engine/src/main/java/org/terasology/rendering/world/WorldRenderer.java

```diff
@@ -33,7 +32,7 @@
     void onChunkUnloaded(Vector3i chunkPos);
 
     public enum WorldRenderingStage {
-        DEFAULT,
+        MONO,
         OCULUS_LEFT_EYE,
         OCULUS_RIGHT_EYE
```

Comment:
Potentially we could change these to more generic LEFT_EYE and RIGHT_EYE. In practice the renderer could hardly support other stereo system as it is peppered with oculus-specific code. As I proceed in the refactoring I will have a look at the concept of having a separate Oculus renderer, perhaps inheriting from the mono one. The problem is, as soon as I touch anything in that department I won't be able to test it as I do not have an Oculus device.